src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -254,6 +254,7 @@ public partial class Socket : System.IDisposable
     {
         public Socket(System.Net.Sockets.SafeSocketHandle handle) { }
         public Socket(System.Net.Sockets.AddressFamily addressFamily, System.Net.Sockets.SocketType socketType, System.Net.Sockets.ProtocolType protocolType) { }
+        [System.Runtime.Versioning.MinimumOSPlatformAttribute("windows7.0")]
```

Comment:
@dotnet/ncl @adamsitnik this was missed in the PR that marked windows specific APIs.